projects/core/utils/format/format-phone.ts

```diff
@@ -18,6 +18,8 @@ export function formatPhone(
 
     let result = countryCode;
 
+    countryCode = countryCode.replace(/[()]/g, '');
```

Comment:
it fixes cases with country code with brackets like +1(264) (Anguilla)

Reply:
What about result above? Should it retain brackets?

Reply:
Yes, the result must be with brackets. I am replacing brackets in countryCode because below  there is a code where value is always without brackets and countryCode is not. (`value.startsWith (countryCode)`)